hdrass/CCW+Rotator_integration_testing/Rotator_interlock_test.ipynb

```diff
@@ -0,0 +1,1007 @@
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await salobj.set_summary_state(mtm, salobj.State.DISABLED)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await salobj.set_summary_state(mtm, salobj.State.ENABLED)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#await salobj.set_summary_state(mtm, salobj.State.OFFLINE)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# CCW check:\n",
+    "The loop is automatically started when the CSC is enabled, but it will fail if it can’t get data from the rotator. The recommended way to bring up MTRotator and MTMount is to first put both in DISABLED state (in any order), then enable MTMount (so the CCW centers itself on the rotator) then enable MTRotator. (Thanks Russell for the explanation)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "CCWstate = await mtm.evt_cameraCableWrapFollowing.aget(timeout=10.)\n",
+    "print(\"CCWstate:\",CCWstate.enabled)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "CCWstate = await mtm.evt_cameraCableWrapFollowing.aget(timeout=10.)\n",
+    "print(\"CCWstate:\",CCWstate.enabled)\n",
+    "await mtm.cmd_enableCameraCableWrapFollowing.set_start(timeout=10.)\n",
+    "CCWstate = await mtm.evt_cameraCableWrapFollowing.aget(timeout=10.)\n",
+    "print(\"CCWstate:\",CCWstate.enabled)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Move to 0 deg Starting Position"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#dir(rot)\n",
+    "rot.cmd_move?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"Test Start\")\n",
+    "\n",
+    "now = datetime.now()\n",
+    "script.evt_logMessage.set_put(level=logging.INFO+1,\n",
+    "                              message=f\"START - {test_message} - Move to 0 deg Starting Position - {now} UTC\")\n",
+    "\n",
+    "print(\"Move to 0.0 deg starting position\")\n",
+    "\n",
+    "await rot.cmd_configureVelocity.set_start(vlimit=1.0)\n",
+    "await rot.cmd_move.set_start(position=1.0,timeout=30.)\n",
+    "#await rot.cmd_moveConstantVelocity.start(timeout=30.)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "alt = 45. * u.deg\n",
+    "az = 0. * u.deg\n",
+    "rot_tel = Angle(0, unit= u.deg) \n",
+    "\n",
+    "target_name=\"Rotator test\"\n",
+    "time_data = await mtptg.tel_timeAndDate.next(flush=True, timeout=2)\n",
```

Comment:
Why use MTPtg instead of AstroPy? Both work, but I'd not use a CSC for current time if I can get it directly. Unless you fear the clock in the machine running your notebook is off?

Reply:
Yes, this seems to be a bit of an overkill.  This code came from another notebook... I keep it for the moment since test were done with MTPtg.